site.yml.sample

```diff
@@ -37,61 +37,61 @@
       when: ansible_distribution == 'Fedora' and ansible_distribution_major_version|int >= 23
 
 - hosts: mons
-  gather_facts: false
+  gather_facts: True
```

Comment:
Interesting, I think I might have ran into this bug then: https://github.com/ansible/ansible/issues/14228

Without this change ``ansible_os_family`` is not defined and the plays fail. When installing packages ``ceph-installer`` is using the ``package-install`` tag, so that might have been what triggered this bug.

Would you prefer tagging the tasks above with ``always``?